test/functional/dd-settlement-only-spec.js

```diff
@@ -130,7 +130,7 @@ fdescribe( 'The dynamic financial aid disclosure', function() {
     expect( page.defaultRateLink.getAttribute( 'href' ) ).toEqual( 'http://nces.ed.gov/collegenavigator/?id=182111#fedloans' );
   } );
 
-  it( 'should open the link to the College Navigator cohort loan default rates in a new tab with the loan default rates section open', function() {
+  fit( 'should open the link to the College Navigator cohort loan default rates in a new tab with the loan default rates section open', function() {
```

Comment:
intentional? If so, let's add a comment to explain.

Reply:
Unintentional! Thanks for the catch! New commit fixes this.